OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -495,7 +500,12 @@ public async Task UpdateWorkshop_WhenModelIsValid_ShouldReturnOkObjectResult()
     public async Task UpdateWorkshop_WhenModelIsInvalid_ShouldReturnBadRequestObjectResult()
     {
         // Arrange
-        workshopServiceMoq.Setup(x => x.Update(workshopUpdateDto)).ReturnsAsync(workshopUpdateDto);
+        //workshopServiceMoq.Setup(x => x.Update(workshopUpdateDto))
+        //    .ReturnsAsync(Result<WorkshopBaseDto>.Failed(new OperationError
+        //    {
+        //        Code = HttpStatusCode.BadRequest.ToString(),
+        //        Description = "Workshop not found.",
+        //    }));
```

Comment:
(Commented code) Please don't leave commented out code. For history you could use git. For knowledge sharing - you should add more explanations than just a bare code.

Reply:
Fixed. Thank you!